content/zh/docs/concepts/architecture/garbage-collection.md

```diff
@@ -0,0 +1,181 @@
+---
+title: Garbage Collection
```

Comment:
I'm not sure but I think this title  should be localized

Reply:
> 

I have modified it